Stack/Opc.Ua.Core/Schema/Opc.Ua.Types.bsd

```diff
@@ -124,7 +124,7 @@
     <opc:Field Name="AdditionalInfoSpecified" TypeName="opc:Bit" />
     <opc:Field Name="InnerStatusCodeSpecified" TypeName="opc:Bit" />
     <opc:Field Name="InnerDiagnosticInfoSpecified" TypeName="opc:Bit" />
-    <opc:Field Name="Reserved1" TypeName="opc:Bit" Length="1" />
+    <opc:Field Name="Reserved1" TypeName="opc:Bit" Length="2" />
```

Comment:
Please revert this manually, it comes incorrect out of code generation...

Reply:
Right, I forgot about that one...
I'll revert it now.

Reply:
I will supply a unit test for this, then it will be an easy catch ...